content/docs/CLI/propose-downstream.md

```diff
@@ -1,13 +1,13 @@
 ---
```

Comment:
please write somewhere in this file that the command used to be named `propose-update`

Reply:
done